client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-welcome-banner/index.tsx

```diff
@@ -47,16 +47,21 @@ export default function SiteWelcomeBanner( {
 
 	const isDismissed = preference?.dismiss;
 	const hideBanner = ! isDashboardView && homePagePreference?.view;
+	const viewDate = preference?.view_date;
 
 	useEffect( () => {
 		if ( ! isDismissed && ! hideBanner ) {
-			savePreferenceType( 'view' );
+			savePreferenceType( 'view', true );
+			if ( ! viewDate ) {
+				savePreferenceType( 'view_date', Date.now() );
+			}
 			handleTrackEvents(
 				isDashboardView
 					? 'calypso_jetpack_agency_dashboard_home_page_banner_view'
 					: 'calypso_jetpack_agency_dashboard_other_page_banner_view'
 			);
 		}
+		// We only want to run this once
```

Comment:
Thanks so much for adding this comment! As someone who's unfamiliar, it helps me understand the intended behavior.